File: src/tasks_java/BankAccount.java
package tasks_java;

public class BankAccount {
    private double balance;
    private int accountNumber;

    BankAccount(double balance, int accountNumber) {
        this.balance = balance;
        this.accountNumber = accountNumber;
    }

    void getBalance() {
        String balanceL = String.format("%.2f", balance);
        System.out.println("Баланс на счете: " + balanceL);
    }

    void deposit(double amount) {
        if (amount > 0) {
            this.balance += amount;
        }
    }

    void withdraw(double amount) {
        if (balance - amount >= 0 && amount > 0) {
            this.balance -= amount;
        } else {
            System.out.println("Недостаточно денег на счете!");
        }
    }
}

File: src/tasks_kotlin/BankAccount.kt
package tasks_kotlin

class BankAccount (
    private var balance: Double,
    private val accountNumber: Int
) {
    fun getBalance() {
        val balance = String.format("%.2f", balance)
        println("Баланс на счете: $balance")
    }

    fun deposit(amount: Double) {
        if (amount > 0) {
            balance += amount
        }
    }

    fun withdraw(amount: Double) {
        if (balance-amount>=0 && amount > 0) {
            balance-=amount
        } else {
            println("Недостаточно денег на счете!");
        }
    }
}